docs/layers/accounts/deploy-accounts.mdx

```diff
@@ -52,33 +52,29 @@ This step-by-step process outlines how to deploy AWS accounts using `atmos` work
 
     Before performing the "Deploying Accounts" step, the root account needs to be configured as an AWS Organization.
 
-     To deploy all accounts, we need to request an increase of the Account Quota from AWS support, which requires an AWS Organization to be created first.
-
-     This process also enables [AWS RAM for Organizations](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_enable-ram.html) via a CLI command, which is required for connecting the Organization.
+    This process also enables [AWS RAM for Organizations](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_policies_enable-ram.html) via a CLI command, which is required for connecting the Organization.
   </Step>
 
   <Step>
     ## <StepNumber/> Raise Account Limits
 
-    Complete the root account setup with the following ClickOps steps.
-
-    From the `root` account (not `SuperAdmin`):
+    To deploy all accounts, we need to request an increase of the Account Quota from AWS support, which requires an AWS Organization to be created first.
 
-    1. Increase the [account quota to 20+](https://us-east-1.console.aws.amazon.com/servicequotas/home/services/organizations/quotas) for the Cloud Posse reference architecture, or more depending on your business use-case
+    From the `root` account (not `SuperAdmin`), increase the [account quota to 20+](https://us-east-1.console.aws.amazon.com/servicequotas/home/services/organizations/quotas) for the Cloud Posse reference architecture, or more depending on your business use-case
 
   </Step>
 
   <Step>
     ## <StepNumber/> Deploying Accounts
 
-    <Note title="IMPORTANT">
+    <Note title="important">
```

Comment:
```suggestion
    <Note title="Important">
```

Reply:
I've made all of these all lowercase. Do you want me to change them all to first letter caps?